lib/Corres_UL.thy

```diff
@@ -104,6 +104,17 @@ lemma corres_gets[simp, corres_no_simp]:
   (\<forall> s s'. P s \<and> P' s' \<and> (s, s') \<in> sr \<longrightarrow> r (a s) (b s'))"
   by (simp add: simpler_gets_def corres_singleton)
 
+lemma corres_gets_return:
```

Comment:
Should I give this one the attributes `[simp, corres_no_simp]`, like `corres_gets` above?

Reply:
Better not to make it `simp`. I would in fact want to remove the `simp` from `corres_gets` if that wouldn't break so many proofs.